partcad/src/partcad/project_factory.py

```
@@ -46,7 +56,9 @@ def __init__(self, ctx, parent, import_config_obj):
 
     def _create(self, config):
         # TODO(clairbee): Finalize the config object if necessary
-        self.project = p.Project(self.ctx, self.name, self.path, include_paths=self.include_paths)
+        self.project = p.Project(
+            self.ctx, self.name, self.path, include_paths=self.include_paths, inherited_config=self.inherited_config
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)